about.html

```diff
@@ -0,0 +1,140 @@
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"
+    defer></script>
+  <script src="https://kit.fontawesome.com/9d6e6aaa13.js" crossorigin="anonymous"></script>
+
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="page-header">
+    <section class="nav-bar">
+      <div class="page-logo">
+        <a href="index.html" class="navbar-brand">
+          <img width="60px" height="80px" src="img/main-icon.png" alt="icon">
+          World Snooker Championship 2022</a>
+      </div>
+      <nav class="nav-container">
+        <button id="menu-btn">
+          <i class="fa-solid fa-bars"></i>
+        </button>
+        <ul class="nav-menu">
+          <li><a href="index.html">Home</a></li>
+          <li class="active">
+            <a href="#">About</a>
+          </li>
+          <li><a href="#program-section">Program</a></li>
+          <li><a href="#player-section">Players</a></li>
+          <li><a href="#partner-section">Partner</a></li>
+          <li class="nav-btn"><a href="#events-section">Events</a></li>
+        </ul>
+      </nav>
+    </section>
+  </header>
+  <main>
+    <article class="headline-container">
+      <section class="headline about-headline">
+        <h1 id="main-heading" class="text-center ms-md-5 me-md-5 ps-md-5 pe-md-5">
+          <span class="greeting text-center"> "Hello! Snooker World" </span>
+          World Snooker Championship 2022
+        </h1>
+        <p class="p-4 bg-white">
+          You’ve probably heard the term ‘Race to the Crucible’ mentioned at some points throughout any given season.
+          This refers specifically to the race to be in the Top 16 as these individuals in the rankings automatically
+          qualify for the Crucible and avoid underdoing the stress of qualifying. Once you hear what the qualifiers have
+          to go through just to get to the main stage, you’ll understand why being in and around the Top 16 of the world
+          rankings is a desirable prospect for all professionals. It’s also why the tournament before the WSC is such an
+          important event for those players. The Top 16 are then seeded, with the defending champion holding the No. 1 seed, and the rest based off their
+          world rankings. None of these players will contest in the first round. The Top 16 face off against the 16
+          individuals that came through the qualifying rounds. As for those 16 that survived the enduring stages of
+          qualifying, this is what they have to go through.
+        </p>
+        <p class="ps-5 pe-5 text-center">
+          Please contact us per Email for any further questions about World Snooker Championship 2022 <br />
+          <a href="#" class="text-center">worldsnooker2022@gmail.com</a>
+        </p>
+      </section>
+    </article>
```

Comment:
- [ ] here again the right semantic tag is `section` tag as these are sections of the page